manifests/plugin/puppetdb/params.pp

```diff
@@ -31,6 +31,9 @@
       fail("${::hostname}: puppetdb_foreman does not support osfamily ${::osfamily}")
     }
   }
-  $address = 'https://localhost:8081/v2/commands'
-  $dashboard_address = 'http://localhost:8080/dashboard'
+  $address = 'https://localhost:8081/pdb/cmd/v1'
+  $dashboard_address = 'http://localhost:8080/pdb/dashboard'
+  $ssl_ca_file       = $::foreman::params::client_ssl_ca
```

Comment:
Also needs an `include foreman::params` in this class to ensure the parameters are defined (evaluation order may vary).